### ISSUE TITLE
Document diff review panel a11y

### ISSUE THREAD

**Issue by derekriemer** [microsoft/vscode#128865]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:
- OS Version:

Steps to Reproduce:

1.
2.
In https://github.com/microsoft/vscode/issues/29496, A lot of progress was made on diff accessibility. Unfortunately this isn't mentioned in the documentation for accessibility. I realized other vscode users are likely missing out on how to use the diff panel and that it has a11y because they don't know about this, and because its not obvious to press f7. Some users might try searching the command pallet but I'm not really sure what is listed there. I think a small note should be added to the vscode documentationn talking about how diff panels can be used.
